<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMS</class>
 <widget class="QDialog" name="DialogMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手动求解</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>原题：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>检查</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_0">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ａ＝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_0">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>56</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>86</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｂ＝</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｃ＝</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>116</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｄ＝</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｅ＝</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｆ＝</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｇ＝</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｈ＝</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｉ＝</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ｊ＝</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>您的答案：</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>59</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout1">
    <item row="0" column="0">
     <widget class="QLabel" name="lblFrom">
      <property name="text">
       <string>None.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>59</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout2">
    <item row="0" column="0">
     <widget class="QLabel" name="lblTo">
      <property name="text">
       <string>None.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
